src/bentoml/_internal/container/__init__.py

```diff
@@ -298,6 +298,11 @@ def health(backend: str) -> bool:
     Args:
         backend: The name of the backend.
 
+    .. note::
+
+        If given backend is a type of OCIBuilder, and the backend is not registered,
+        make sure to register it with ``bentoml.container.register_backend``.
```

Comment:
Can we just use registries on subclass like we do for schedulers / IO descriptors?

Reply:
Yep. We can. I will make a separate PR for this.